tests/install.sh

```diff
@@ -10,6 +10,9 @@
 npm install --quiet -g jshint 1>/dev/null
```

Comment:
It might be better to move all the npm packages to package.json. That would be our first step towards separating the AutolabJS testsuite from the rest of the containerized application.

Reply:
We also get the advantage of managing the version numbers of these dependencies.